lib/ash/actions/read/read.ex

```diff
@@ -609,6 +609,9 @@ defmodule Ash.Actions.Read do
                     end)
                   end)
               end
+
+            nil ->
```

Comment:
Hmmmmmm how are we even getting to `nil` here... I guess we're unloading a calculation or som thing somewhere?

Reply:
I haven't had much time to dig deeper. I am still working on upgrading to 3.0, and I'm already over my allotted time budget